tpp/src/main/Member.js

```diff
@@ -78,13 +79,21 @@ export default class Member extends CoreMember {
      * Creates a representable that acts as another member via an access token.
      *
      * @param accessTokenId - ID of the access token
+     * @param {CustomerTrackingMetadata} customerTrackingMetadata
      * @return new member that acts as another member
      */
-    forAccessToken(accessTokenId: string): Representable {
+    forAccessToken(accessTokenId: string, customerTrackingMetadata: CustomerTrackingMetadata): Representable {
```

Comment:
Could you also add the customer initiated flag here? (mirroring the java sdk)
Basically the customer tracking metadata would only make sense if the request is initiated by the customer.